backend/src/main/java/de/neuefische/backend/kunden/customer.java

```diff
@@ -0,0 +1,21 @@
+package de.neuefische.backend.kunden;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class customer {
+    private String id;
+    private String firstname;
+    private String surnname;
+    private String adress;
+    private String postalCode;
+    private int phonenumber;
```

Comment:
Are you sure, an int would be enough? Won't you need a long?

Reply:
Right!!!!